### ISSUE TITLE
Pretraining/Finetuning stage of FFVI and Cannot reproduce quantitative evaluation

### ISSUE THREAD

**Issue by MaureenZOU** [amjltc295/Free-Form-Video-Inpainting#30]
Hi Ya-Liang, after going through the issues, you have mentioned in https://github.com/amjltc295/Free-Form-Video-Inpainting/issues/20 that there are pretraining and finetuning stage of your model. Also, when I train your model from scratch, I found your GAN loss is set to zero. Could you please explain the detailed training schedule of the pretraining and finetuning stage and loss type/weight? Thanks in advance : )

**Comment by MaureenZOU** [comment]
Sorry to bother you again, there is an additional question. How do you exactly do the quantitative evaluation in paper?

e.g. You have mentioned that the result is evaluate on FVI testing set. Your algorithm could only take a window size of k, in this way, how do you define the predicted image, is it the center frame of k? Or you split the whole video to n/k split and each of the output count? And does your evaluation span across the whole video length? Thanks!

Xueyan

**Comment by MaureenZOU** [comment]
![Screen Shot 2020-07-14 at 1 51 28 PM](https://user-images.githubusercontent.com/11957155/87475879-5e11f900-c5da-11ea-8fb0-33867689bf54.png)

As you have mentioned in the Caption, Quantitative comparison is on FVI testing set without border. Could you please share the border length? Thanks a lot!

**Comment by MaureenZOU** [comment]
First question solved, found in SUP. Thanks!

**Comment by amjltc295** [comment]
This table is an average score of different mask-to-frame ratios (please refer to the supplementary materials). I'm not sure which ratio you're using, but it should be a ~60% one considering the score.

**Comment by MaureenZOU** [comment]
Thanks a lot for your information! I am really sorry to have an additional question : (
After training your model around 300 epochs with the default setting. The results are actually very fluctuated, could you please verify whether it makes sense in the attached file. Each row represent an epoch, and it is ordered in chronological order.

**Comment by amjltc295** [comment]
Did you include the discriminator loss? This is normal if you only use perceptual loss. The pretraining step is done if the loss is converged. Then you need to include the discriminator. If it's still like this after fine-tuning, you will probably need to tune some hyper parameters.

**Comment by MaureenZOU** [comment]
Thanks a lot! I only use perceptual loss. I will finetune with discriminator loss to see what happens. Thanks!

**Comment by MaureenZOU** [comment]
Thanks for your code again! I have reproduce all the results on your paper using the model provided. At the same time, reproduce the paper number by training the model myself. It is a nice repo with clean code : )

**Comment by amjltc295** [comment]
No problem, thank you for verifying it as well :)
